bots/tests-scan

```diff
@@ -118,6 +121,11 @@ def main():
                         help='Test contexts to use.')
     parser.add_argument('-p', '--pull-numbers', action="append", default=[ ],
                         help='Pull requests to scan for tasks')
+    parser.add_argument('--amqp', action="store_true", default='localhost',
+                        help='The url of the amqp server to publish to.')
+    parser.add_argument('--queue', action="store_true", default='tests',
+                        help='The name of the queue to consume from.')
```

Comment:
Same thing again, these are string value, not boolean options. Also, this is not "consume from", but "publish to".